apps/web/src/components/molecules/CompanyFiltersContainer/CompanyFiltersContainer.tsx

```diff
@@ -0,0 +1,89 @@
+import type { EmploymentCheckbox } from "@/components/views/JobListingView/JobListingView"
+import type { Dispatch, FC, SetStateAction } from "react"
+
+interface CompanyFiltersContainer {
+  chosenLocation: string
+  setChosenLocation: Dispatch<SetStateAction<string>>
+  searchName: string
+  setSearchName: Dispatch<SetStateAction<string>>
+  chosenEmployments: EmploymentCheckbox[]
+  setChosenEmployments: Dispatch<SetStateAction<EmploymentCheckbox[]>>
+  chosenSort: string
+  setChosenSort: Dispatch<SetStateAction<string>>
+}
+
+const places = ["Alle", "Oslo", "Bergen", "Trondheim", "Tromsø", "Fredrikstad", "Annet"]
```

Comment:
What do you think about passing down these as props based on the set of locations that exist in the fetched job listings? We can probably keep "Alle" here and just concat the locations from the props?